meta-refkit/recipes-ros/cv-bridge/cv-bridge_%.bbappend

```diff
@@ -0,0 +1,3 @@
+# TODO: This way cv_bridge doesn't require libboost_python, but this tweak
+#       should be dropped after ROS is compatible with python3-only distro.
+EXTRA_OECMAKE += "-DANDROID=ON"
```

Comment:
Hmm, the next patch says: `switch ROS to python3 `. Is it python3-compatible or not?